obs-studio-server/source/osn-volmeter.cpp

```diff
@@ -35,9 +35,10 @@ osn::VolMeter::VolMeter(obs_fader_type type)
 		throw std::exception();
 }
 
-osn::VolMeter::~VolMeter()
```

Comment:
Fixes this shutdown crash https://sentry.io/streamlabs-obs/obs-server/issues/695709152/?query=is%3Aunresolved